app/components/Views/SendFlow/Confirm/index.js

```diff
@@ -561,17 +562,21 @@ class Confirm extends PureComponent {
 		);
 	};
 
-	handleSetGasFee = (customGas, customGasPrice) => {
+	handleSetGasFee = (customGas, customGasPrice, warningGasPriceHigh, details) => {
 		const { prepareTransaction, transactionState } = this.props;
 		let transaction = transactionState.transaction;
 		transaction = { ...transaction, gas: customGas, gasPrice: customGasPrice };
 		prepareTransaction(transaction);
+		warningGasPriceHigh
+			? this.setState({ warningGasPriceHigh })
+			: this.setState({ warningGasPriceHigh: undefined });
```

Comment:
you could do 
```suggestion
			this.setState({ warningGasPriceHigh });
```

since warningGasPriceHigh could be undefined ?

Reply:
Updated